cl.conf

```diff
@@ -50,6 +50,20 @@ systemd:
         RemainAfterExit=true
         User=core
 
+        [Install]
```

Comment:
It seems we can use cl syntax to disable locksmithd.
https://coreos.com/os/docs/latest/update-strategies.html
```
systemd:
  units:
    - name: update-engine.service
      mask: true
    - name: locksmithd.service
      mask: true
```

Also, if we do 'vagrant halt' then 'vagrant up', does it enable a disabled locksmithd service?

Reply:
I think the `RemainAfterExit=true` is supposed to keep the state throughout the vm lifetime ?

Reply:
@amattyang, kops doesnt allow masking a service in this method at the moment.  You can disable, but not mask.  I was keeping it the same as the kops method. Could switch it up here.

No, either way on 'vagrant halt' then 'vagrant up' locksmithd will not be started, as this masks the service and doesnt just disable it.  Recall disable only stops the service, until the next reboot.  

@mordr, I dont think it matters in this case. RemainAfterExit is how systemd treats the status of the service after ExecStart exits. If true, then after the ExecStart, the status of the service is kept as active. If left at the default (false) then after ExecStart exits, on a oneshot, the status is set to inactive.

RemainAfterExit is useful if you want to keep the oneshot service active, so you can **stop** the service at a later date, which could run an **ExecStop**. i.e in case you wanted to revert the masking of locksmithd.

Something like
`ExecStop=/usr/bin/systemctl unmask --now locksmithd.service`
would unmask locksmithd when __systemctl stop mask-locksmithd__ is executed

Reply:
ok, thanks.

Reply:
@jaimemurillo16 : let's switch it up to use cl syntax if you don't mind. The result *.ign should be the same.